rgdps/usecases/users.py

```diff
@@ -310,3 +310,32 @@ async def request_status(
         return UserPrivilegeLevel.ELDER_MODERATOR
 
     return UserPrivilegeLevel.NONE
+
+
+async def synchronise_search(ctx: Context) -> bool | ServiceError:
+    users = await repositories.user.all_ids(ctx)
+
+    for user_id in users:
+        user = await repositories.user.from_id(ctx, user_id)
+
+        if not user:
+            continue
+
+        await repositories.user.create_meili(ctx, user)
+
+    return True
+
+
+# Is the return type right?
+async def search(
+    ctx: Context,
+    page: int,
+    page_size: int,
+    query: str,
+) -> repositories.user.UserSearchResults | ServiceError:
```

Comment:
is there need to list service error as return type?

Reply:
Consistency, in case in the future this becomes more complex.